bootstrap/090_post_setup.sql

```diff
@@ -34,14 +34,14 @@ BEGIN
 CREATE OR REPLACE TASK TASKS.WAREHOUSE_EVENTS_MAINTENANCE
     SCHEDULE = '60 minute'
     ALLOW_OVERLAPPING_EXECUTION = FALSE
-    USER_TASK_MANAGED_INITIAL_WAREHOUSE_SIZE = "XSMALL"
+    USER_TASK_MANAGED_INITIAL_WAREHOUSE_SIZE = "LARGE"
```

Comment:
I think this one can stay xsmall. It appears to complete very quickly even on xsmall